backend/lib/app.js

```diff
@@ -1,3 +1,5 @@
+process.title = "validService";
```

Comment:
Maybe "validatorService"? There are a lot of valid services out there, I guess.

Reply:
There's a limit of 15 characters for process names, i.e. the value here will get truncated. With the suggestion of `validatorService` we'd need to adjust the `pkill` target to `validatorServi` to still work properly. I'd like to avoid such a mismatch of corresponding strings.

However, I do agree that the semantics of this is off. I'm happy to change it to sth. like `validatorSrvce` or `vldtrService` if you like that better.

Reply:
After discussion with @domachine it should be possible to remove the `pkill` part completely and assigning a process name would not even be necessary. However, now the action fails due to something that seems unrelated to that. I'm investigating.

Reply:
The default port changed to 8082 in https://github.com/secvisogram/csaf-validator-service/commit/2dff1a36bfb53df6099449aa808bf5b6641466de
The tests still checked for the server running on 3000. I changed the port for tests to 8082 as well and it's working again.
The process name is removed completely.